### ISSUE TITLE
Directions on how to answer

### ISSUE THREAD

**Issue by onkase** [toasterco/Quiz-Master-for-Google-Meet#3]
I was just wondering how the students answer.  Do they get selection through quiz master or do they do it in the chat?

**Comment by skezo** [comment]
@onkase thanks for the question.

There are two types of users:
1. The quiz master. (indicated by the "crown" icon 
2. The players/students. 

![Screenshot 2020-10-29 at 8 35 59 AM](https://user-images.githubusercontent.com/793287/97511321-c803b200-19c1-11eb-8b88-4b4d6a37a9fc.png)

The quiz master can type in a question and then press "Start question":

![Screenshot 2020-10-29 at 8 43 34 AM](https://user-images.githubusercontent.com/793287/97511798-e5854b80-19c2-11eb-9b93-7425c0b3100e.png)


When the quiz master presses "start question" the players/students can then see the question and the "Answer" button is enabled.

![Screenshot 2020-10-29 at 8 39 48 AM](https://user-images.githubusercontent.com/793287/97511558-4fe9bc00-19c2-11eb-932a-a9af3901e80f.png)

The players/students can then answer the question by pressing "answer". The first person to press "answer" will be highlighted in green on the list:
![Screenshot 2020-10-29 at 8 46 10 AM](https://user-images.githubusercontent.com/793287/97511943-3b59f380-19c3-11eb-9433-60c9aacc30cb.png)

Let me know if you need any more information or have any suggestion how it can be improved. 

**Important:** Each participant requires the extension to participate.

**Comment by onkase** [comment]
So they answer by speaking or tying in the answer - I wasn’t able to see
the participant side but they said they were not able to type anything -
what about multiple choice questions?

**Comment by skezo** [comment]
> So they answer by speaking or tying in the answer

They answer by speaking. 

> but they said they were not able to type anything -

Did the participants/students have the extension installed?

> what about multiple choice questions?

These would be typed out. Similar to other questions e.g.
`Which type of musical composition is also an Olympic figure skating jump? A. Sarabande B. Bagatelle C. Mazurka`
We are thinking to update the input area to support better formatting. So instead you could write it like:
```
Which type of musical composition is also an Olympic figure skating jump?
A. Sarabande
B. Bagatelle
C. Mazurka
```

**Comment by onkase** [comment]
The student had it installed but was only able to press the answer button
without any type of input box

**Comment by skezo** [comment]
@onkase Did the student who pressed it first highlight in green (similar to below screen shot)? 
![Screenshot 2020-10-29 at 8 46 10 AM](https://user-images.githubusercontent.com/793287/97511943-3b59f380-19c3-11eb-9433-60c9aacc30cb.png)

**Comment by onkase** [comment]
Yes, I am still testing it out so I will ask my class a few more times to
see if it works.

**Comment by skezo** [comment]
@onkase 
Great. Let us know if you have any feature request or ideas on how to improve it.

**Comment by skezo** [comment]
@onkase 
We have pushed an updated (v0.6.8) to support multiline questions. This should help with multiple choice type questions. For example this:

`Which type of musical composition is also an Olympic figure skating jump? A. Sarabande B. Bagatelle C. Mazurka`

can now be typed out as:

```
Which type of musical composition is 
also an Olympic figure skating jump?

A. Sarabande
B. Bagatelle
C. Mazurka
```

You should see this update on your computer in the next few days as it propagates.
